src/routes/InventoryDetail.js

```diff
@@ -55,7 +55,7 @@ const Inventory = ({ entity, currentApp, clearNotifications, loadEntity }) => {
             }}
         >
             <PageHeader className={classnames('pf-m-light ins-inventory-detail', additionalClasses)} >
-                <Breadcrumb>
+                <Breadcrumb ouiaId="inventory">
```

Comment:
Not sure about this!
Whether to put here or in breadcrumb_items or its not required

Reply:
I would change it from `inventory` to `systems-list` as that's what the previous page shows.